docs/agent/secrets.md

```diff
@@ -196,11 +201,35 @@ will refuse to use it otherwise):
 - Have at least `exec` right for the owner.
 
 Also:
-- The executable will be run with an empty environment.
+- The executable will share the same environment variables than agent.
+- Never output sensitive information on STDERR. If the binary exit with a
+  different status code than `0` the agent will log the standard error output
+  of the executable to ease troubleshooting.
+
+#### Windows
+
+On Windows, the executable set as `secret_backend_command` **MUST** (the agent
+will refuse to use it otherwise):
```

Comment:
```suggestion
refuses to use it otherwise):
```

Reply:
I think `will refuse` is correct in this case.